### PR TITLE
Extend Parsed Variables

### PR DESCRIPTION
**Pull request by tkukuljan**
Bring in the place holder for the functionality of the split of the coverage units into INV and INS component.

### CODE REVIEW

File: ifrs17/Import/Importers.ipynb [hunk 1]
```diff
@@ -1458,6 +1458,7 @@
                 "\n        }, ImportFormats.Cashflow",
                 "\n    ).WithTarget(workspace).ExecuteAsync();",
                 "\n    ",
+                "\n    await workspace.CheckForCompositeAmountTypeAndNormalize(AmountTypes.CU);",
```
Comment (dcolleoni) [inline ifrs17/Import/Importers.ipynb:1461]: give the storage as input. The storage should provide all it is needed (hierarchyCache)

Comment (tkukuljan) [reply ifrs17/Import/Importers.ipynb]: implemented

File: ifrs17/Constants/Validations.ipynb [hunk 2]
```diff
@@ -124,6 +124,8 @@
                 "\n    (Error.AocTypePositionNotSupported    , 1) => $\"The position of the AoC Type {s[0]} is not supported.\",",
                 "\n    (Error.AocConfigurationOrderNotUnique , _) => $\"Two or more AoC Configurations have the same Order.\",",
                 "\n    (Error.AccidentYearTypeNotValid       , 1) => $\"The parsed AccidentYear {s[0]} is invalid. Expected Accident Year input of type int.\",",
+                "\n    (Error.CoverageUnitComponentNotFound  , 2) => $\"Coverage Units component {s[0]} is missing for Data Node {s[1]}.\",",
```
Comment (dcolleoni) [inline ifrs17/Constants/Validations.ipynb:127]: they should be defined in the client project

File: ifrs17/Utils/ImportCalculationMethods.ipynb [hunk 3]
```diff
@@ -409,6 +409,34 @@
...
+                "public static async Task<ActivityLog> ExtendParsedCoverageUnits (this IWorkspace workspace, Systemorph.Vertex.Hierarchies.IHierarchicalDimensionCache hierarchyCache) {}"
+            ],
+            "metadata": {},
+            "execution_count": 0,
+            "outputs": []
+        },
+        {
+            "cell_type": "code",
+            "source": [
+                "public static async Task ExtendParsedVariables (this IWorkspace workspace, Systemorph.Vertex.Hierarchies.IHierarchicalDimensionCache hierarchyCache) =>",
```
Comment (dcolleoni) [inline ifrs17/Utils/ImportCalculationMethods.ipynb:433]: replace hierarchyCache with the ParsingStorage

**Review by dcolleoni — APPROVED** [review]

**Pull request merged**